### PR TITLE
Add countdown for leap 42.3

### PR DESCRIPTION
**Pull request by bisht-richa**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -133,18 +133,17 @@ <h4 class="wow fadeIn" data-wow-delay=".6s" lang="en">Get the most complete Linu
...
           </div>
           <div class="opensuse-countdown__call-to-action">
-            <span lang="en">for openSUSE Conference</span>
+            <span lang="en">for Leap 42.3</span>
           </div>
         </a>
       </div>
       <div class="opensuse-countdown__released hidden">
-        <a href="https://media.ccc.de/c/osc17" target="_blank">
+        <a href="https://www.youtube.com/user/opensusetv" target="_blank">
```
Comment (cyntss) [inline index.html]: Did they require a link to youtube? normally it is to the download button. Check in the history of this file

Comment (cyntss) [inline index.html]: Check here, this is what you need:

https://github.com/openSUSE/landing-page/blob/b67bd7b60ef3df370ee2df61616f6b7ef3f80f8d/index.html

Comment (ddemaio) [inline index.html]: @cyntss the youtube link is for the openSUSE Conference videos. Right now we don't have a date for the release of 42.3 other than end of July. @lnussel might have a better idea about the release next week, which at that point we would want to change it to the new software.o.o link https://software.opensuse.org/distributions/leap

Comment (bisht-richa) [inline index.html]: I have replaced youtube link with this https://software.opensuse.org/distributions/leap.

**Comment by kwwii** [comment]
@bisht-richa, @cyntss, @lnussel:  Is this good to go?

**Comment by lnussel** [comment]
we may need to tune the link wrt software.o.o but I can do that myself. otherwise fine with me.

**Pull request merged**